python/cuml/internals/api_context_managers.py

```diff
@@ -100,6 +134,19 @@ def set_api_memory_type(mem_type):
 
 
 def set_api_output_dtype(output_dtype):
+    """
+    Sets the output data type for the current internal API context.
```

Comment:
Explain the impact of setting this and what is meant by data type. I would actually stick to the word `dtype`, since it will be clearer to those familiar with numpy/cupy what this is.